catalog.bom

```diff
@@ -1,18 +1,50 @@
 brooklyn.catalog:
-  id: jmeter
-  displayName: Apache JMeter
-  version: 0.9.0
-  iconUrl: https://upload.wikimedia.org/wikipedia/commons/2/22/Apache_JMeter.png
-  license: Apache-2.0
-
-  description: |
-    Runs Apache JMeter against a configured endpoint.
-
+  version: "0.1-SNAPSHOT"
   publish:
     license_code: Apache-2.0
     overview: README.md
 
-  item:
-    type: io.cloudsoft.jmeter.JMeterNode
-    name: JMeter Node
+  brooklyn.libraries:
+  - https://oss.sonatype.org/content/repositories/snapshots/io/cloudsoft/jmeter/brooklyn-jmeter/0.11.0-SNAPSHOT/brooklyn-jmeter-0.11.0-20170404.145804-2.jar
+
+  items:
+    - id: jmeter-load-app
+      iconUrl: classpath://io/cloudsoft/jmeter/logo.jpg
```

Comment:
I think it's actually simplest to replace this with a link to the file in GitHub: https://raw.githubusercontent.com/cloudsoft/jmeter-entity/501a3415af373f9071117d8e530e57f728d92154/src/main/resources/io/cloudsoft/jmeter/logo.jpg